go.mod

```diff
@@ -17,11 +17,11 @@ require (
 	github.com/spf13/cobra v1.5.0
 	github.com/spf13/pflag v1.0.5
 	github.com/stretchr/testify v1.8.0
-	github.com/tendermint/tendermint v0.34.20-rc0.0.20220630153554-17f4ea3680de
+	github.com/tendermint/tendermint v0.34.20
```

Comment:
The changes are basically v1 mempool bug fixes, and remove the warning about v1 mempool.

Reply:
Do we use the ABCI socket client? there is a known issue with the V1 mempool. (So the release v0.34.20 note)

Reply:
In v0.7, we actually don't use V1 mempool at all, we just need the tx indexer fix.